Source/KSCrash/Recording/KSCrash.m

```diff
@@ -117,6 +117,59 @@ @implementation KSCrash
 #pragma mark - Lifecycle -
 // ============================================================================
 
++ (void)load
+{
+    [self classDidBecomeLoaded];
+}
+
++ (void)initialize
+{
+    if (self == [KSCrash class]) {
+#if KSCRASH_HAS_UIAPPLICATION
+        NSNotificationCenter* nCenter = [NSNotificationCenter defaultCenter];
```

Comment:
Lets move this into a method and call it from here

Reply:
Done